assets/js/base/components/tabs/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { withInstanceId } from '@wordpress/compose';
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const TabButton = ( {
+	tabId,
+	onClick,
+	children,
+	selected,
+	ariaLabel,
+	...rest
```

Comment:
It's a common pattern to use the term `...rest` to refer to the "rest of the props".  I also [copied this from Gutenberg core](https://github.com/WordPress/gutenberg/blob/4889933f491a91e76af7de01364c4c506377fd37/packages/components/src/tab-panel/index.js#L19) (which doesn't justify using it, just pointing to it being a relatively [common usage pattern](https://github.com/WordPress/gutenberg/search?l=JavaScript&q=%27...rest%27).  This particular component (`TabButton`) isn't exported but only used internally, so I'm not too concerned about the naming here.

Also, it's possible this tabs component will get modified as a part of #1474 so it can be handled there if still seen to be problematic.